ansys/mapdl/reader/rst.py

```diff
@@ -2741,7 +2741,7 @@ def _plot_point_scalars(self, scalars, rnum=None, grid=None,
             rng = kwargs.pop('rng', None)
 
         return_cpos = kwargs.pop('return_cpos', False)
-        cmap = kwargs.pop('cmap', 'jet')
+        cmap = kwargs.pop('cmap', 'viridis')
```

Comment:
Let's provide the ability to override this and to use the theme parameters. If we hard code this, then it's impossible to change or override.

Reply:
Do you mean we shouldn't hardcode a default?

Reply:
We're good for now, we can have a follow-up that optimizes our use of themes.